client/Mac/MRDPView.m

```diff
@@ -960,99 +917,80 @@ BOOL mac_pre_connect(freerdp* instance)
 	settings->OrderSupport[NEG_POLYGON_CB_INDEX] = FALSE;
 	settings->OrderSupport[NEG_ELLIPSE_SC_INDEX] = FALSE;
 	settings->OrderSupport[NEG_ELLIPSE_CB_INDEX] = FALSE;
-	
 	PubSub_SubscribeChannelConnected(instance->context->pubSub,
-					 (pChannelConnectedEventHandler) mac_OnChannelConnectedEventHandler);
-	
+	                                 (pChannelConnectedEventHandler) mac_OnChannelConnectedEventHandler);
 	PubSub_SubscribeChannelDisconnected(instance->context->pubSub,
-					    (pChannelDisconnectedEventHandler) mac_OnChannelDisconnectedEventHandler);
+	                                    (pChannelDisconnectedEventHandler) mac_OnChannelDisconnectedEventHandler);
 
-	if (!freerdp_client_load_addins(instance->context->channels, instance->settings))
+	if (!freerdp_client_load_addins(instance->context->channels,
+	                                instance->settings))
 		return FALSE;
 
-	if (freerdp_channels_pre_connect(instance->context->channels, instance) != CHANNEL_RC_OK)
```

Comment:
@MartinHaimberger Yes, moved to the core library (as every client needs to call that)